### PR TITLE
Bolivia holidays update

### PR DESCRIPTION
**Pull request by KJhellico**

### CODE REVIEW

File: holidays/countries/bolivia.py [hunk 1]
```diff
@@ -121,29 +106,21 @@ def _populate(self, year):
 
         # Pando Day.
         if self.subdiv == "N":
-            self[date(year, OCT, 11)] = "Dia del departamento de Pando"
+            self[date(year, OCT, 11)] = "Día del departamento de Pando"
```
Comment (arkid15r) [inline holidays/countries/bolivia.py:109]: This one seems to be a bit off - https://en.wikipedia.org/wiki/Pando_Department
Should we change it to Sep 24?

File: holidays/countries/bolivia.py [hunk 2]
```diff
@@ -121,29 +106,21 @@ def _populate(self, year):
...
-        self[date(year, NOV, 2)] = name
-
-        if self.observed and date(year, NOV, 2).weekday() == SUN:
-            self[date(year, NOV, 2) + td(days=+1)] = f"{name} (Observed)"
+        _add_with_observed(date(year, NOV, 2), "Todos Santos")
 
         # Potosí Day.
         if self.subdiv == "P":
-            self[date(year, NOV, 10)] = "Dia del departamento de Potosí"
+            self[date(year, NOV, 10)] = "Día del departamento de Potosí"
```
Comment (arkid15r) [inline holidays/countries/bolivia.py:116]: We could consolidate the subdiv based holidays below and optimize them with `elif`s.

File: test/countries/test_bolivia.py [hunk 3]
```diff
@@ -10,255 +10,243 @@
...
+            "2021-02-12",
+            "2022-02-25",
+            "2023-02-17",
+            "2024-02-09",
+        ):
+            self.assertNoHoliday(dt)
+            self.assertHolidaysName(name, Bolivia(subdiv="O"), dt)
+
+    def test_carnival(self):
+        dt = (
```
Comment (arkid15r) [inline test/countries/test_bolivia.py:82]: Could be inlined.

File: test/countries/test_bolivia.py [hunk 4]
```diff
@@ -10,255 +10,243 @@
...
+        self.assertHolidaysName("Feriado por Carnaval", dt)
 
     def test_good_friday(self):
-        for dt in (date(2022, 4, 20), date(2021, 4, 1)):
-            self.assertNotIn(dt, self.holidays)
-
-        for dt in (date(2022, 4, 15), date(2023, 4, 7)):
-            self.assertIn(dt, self.holidays)
-            self.assertEqual(self.holidays[dt], "Viernes Santo")
+        dt = (
```
Comment (arkid15r) [inline test/countries/test_bolivia.py:103]: This one too.

**Comment by arkid15r** [comment]
Nice refactoring!

File: holidays/countries/bolivia.py (hunk 1 above)
Comment (KJhellico) [reply holidays/countries/bolivia.py:109]: Seems Oct 11 - it's Battle of Bahia anniversary, and department holiday is Sep 24. Thank you!

**Comment by arkid15r** [comment]
Nicely done!

**Pull request merged**